_haikus/game_of_clouds.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Game of Clouds
+author: Rishab Menon
+---
+
+Iron Throne win who? <br>
+Google, Microsoft, AWS? <br>
```

Comment:
This line has one extra syllable, maybe you can sub `Azure` for Microsoft

Reply:
Yes, sure thing! Thanks for the suggestion :)